UnitTests/BraintreePayPalTests/BTPayPalRequest_Tests.swift

```diff
@@ -96,8 +96,15 @@ class BTPayPalRequest_Tests: XCTestCase {
 
     // MARK: - enablePayPalAppSwitch
 
-    func testEnablePayPalAppSwitch_whenNotPassed_defaultsValueAsFalse() {
-        let request = BTPayPalVaultRequest()
+    func testEnablePayPalAppSwitch_whenInitialized_setsAllRequiredValues() {
+        let request = BTPayPalVaultRequest(
+            userAuthenticationEmail: "fake@gmail.com",
+            enablePayPalAppSwitch: true,
+            universalLink: URL(string: "my-website-is-cool.com")!
+        )
+
+        XCTAssertEqual(request.userAuthenticationEmail, "fake@gmail.com")
         XCTAssertFalse(request.enablePayPalAppSwitch)
```

Comment:
qq here - should we `XCTAssertTrue` instead since we set `enablePayPalAppSwitch` on line 102

Reply:
Yep, we actually weren't setting this on the init, which is why this test was passing. Updated here: https://github.com/braintree/braintree_ios/pull/1232/commits/89fef878b314959db2ccb3aefc83c5527b64eff3